modyn/evaluator/internal/pytorch_evaluator.py

```diff
@@ -0,0 +1,177 @@
+import io
+import logging
+import multiprocessing as mp
+import os
+import pathlib
+import queue
+import traceback
+from inspect import isfunction
+from typing import Any, Callable, Optional, Union
+
+import torch
+from modyn.evaluator.internal.dataset.evaluation_dataset import EvaluationDataset
+from modyn.evaluator.internal.utils import EvaluationInfo, EvaluatorMessages
+
+EVALUATION_TRANSFORMER_FUNC_NAME = "evaluation_transformer"
+LABEL_TRANSFORMER_FUNC_NAME = "label_transformer_function"
```

Comment:
this is the same for the pytorch trainer, right? let's move these constants to common

Reply:
Yes, exactly. I moved them to `modyn.utils`.